test/JDBC/upgrade/13_5/preparation/bbf_view_def_upgrade-vu-prepare.sql

```diff
@@ -0,0 +1,10 @@
+CREATE VIEW view_3135_upd AS SELECT 1;
+GO
+
+CREATE VIEW dep_view_3135 AS SELECT * FROM sys.babelfish_view_def;
```

Comment:
can we create a view on non-existent tables? this table is only available on 14 latest

Reply:
No, it will fail. I will try to remove it in next iteration.

Reply:
Fixed